app/src/main/java/com/example/tasky/feature_authentication/presentation/components/TextFieldComponent.kt

```diff
@@ -0,0 +1,81 @@
+package com.example.tasky.feature_authentication.presentation.components
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Check
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.unit.sp
+import com.example.tasky.R
+import com.example.tasky.presentation.theme.GreenValid
+import com.example.tasky.presentation.theme.TextFieldBackground
+import com.example.tasky.presentation.theme.HintColor
+import com.example.tasky.presentation.theme.InputTextColor
+import com.example.tasky.presentation.theme.VisibilityIcon
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun TextFieldComponent(
```

Comment:
Every composable is a component, therefore I find that doesn't add any information to it. What about `TaskyTextField` instead? That clarifies that it's the default text field in the app's own style